Makefile

```diff
@@ -157,6 +169,9 @@ docu_htmlclean:
 docu_htmlnoheaderclean:
 	rm -rf documentation/htmlnoheader
 
+docu_pdfclean:
```

Comment:
should `docu_clean` call the `docu_pdfclean` so actually deleting any type of doc?

Reply:
Good catch. Thanks.